warp10/src/main/java/io/warp10/GenerateCryptoKey.java

```diff
@@ -16,26 +16,28 @@
 
 package io.warp10;
 
-import io.warp10.continuum.Configuration;
-import org.bouncycastle.util.encoders.Hex;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.security.SecureRandom;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
 
+import org.bouncycastle.util.encoders.Hex;
+
+import io.warp10.continuum.Configuration;
+
 public class GenerateCryptoKey {
 
-  private static final Logger LOG = LoggerFactory.getLogger(GenerateCryptoKey.class);
+  private static final String CLASS_HASH_KEY = "class.hash.key";
+  private static final String LABELS_HASH_KEY = "labels.hash.key";
+  private static final String TOKEN_HASH_KEY = "token.hash.key";
+  private static final String APP_HASH_KEY = "app.hash.key";
+  private static final String TOKEN_AES_KEY = "token.aes.key";
+  private static final String SCRIPTS_AES_KEY = "scripts.aes.key";
+  private static final String METASETS_AES_KEY = "metasets.aes.key";
+  private static final String LOGGING_AES_KEY = "logging.aes.key";
+  private static final String FETCH_HASH_KEY = "fetch.hash.key";
   private static final SecureRandom sr = new SecureRandom();
 
-  private static final Map<String, Integer> keys = new HashMap<String, Integer>();
+  private static final Map<String,Integer> keys = new HashMap<String,Integer>();
```

Comment:
Use a LinkedHashMap so the order is the one we select

Reply:
done